website/src/content/docs/docs/libraries/typespec-client-generator-core/reference/data-types.md

```diff
@@ -57,6 +57,7 @@ enum Azure.ClientGenerator.Core.InitializedBy
 | ------------ | ----- | ------------------------------------------------- |
 | individually | `1`   | The client could be initialized individually.     |
 | parent       | `2`   | The client could be initialized by parent client. |
+| none         | `4`   | The client can't be initialized.                  |
```

Comment:
What does it mean that a client can't be initialized?

Reply:
There is concept "operation group" for Python/JS. Operation group can't be initialized independently and can only be accessed by its parent.

For example, class [Operations](https://github.com/Azure/azure-sdk-for-python/blob/21d8b165afb9497dafcce8702beb6bf5f4c1d881/sdk/trustedsigning/azure-mgmt-trustedsigning/azure/mgmt/trustedsigning/_client.py#L86) is operation group, you can use it by `client.operations.xxx` but it can't be initialized independently by user like `operations = Operations(...)`.